### ISSUE TITLE
Pass kwargs to custom FilterSet class.

### ISSUE THREAD

**Issue by jckw** [graphql-python/graphene-django#800]
When using a DjangoFilterConnectionField it is currently possible to add different arguments whilst also using a custom filter set class:

```
class PostsFilter(django_filters.FilterSet):
    @property
    def qs(self):
        # would like to access some_field here to so some fancy filter stuff
        return super().qs.filter(...)


class Query(ObjectType)
    posts = DjangoFilterConnectionField(
        Post, filterset_class=PostsFilter, some_field=graphene.String()
    )
```

Is it possible to access `some_field` inside of the `qs` method of the custom filterset? Or is there a different recommended way of working with custom (non-field related) parameters and DjangoFilterConnectionField?

**Comment by jkimbo** [comment]
@jckw you should be able to create a custom resolve method and access the arguments that way:

```python
class Query(ObjectType)
    posts = DjangoFilterConnectionField(
        Post, filterset_class=PostsFilter, some_field=graphene.String()
    )

	def resolve_posts(root, info, some_field):
		return DjangoPost.objects.filter(...);
```

Unfortunately, because the qs property on `django_filters.FilterSet` is a property and not a function you can't pass arguments to it.

**Comment by BossGrand** [comment]
closing this, repopen if the solution given doesn't work

**Comment by jckw** [comment]
Quite a while since I opened this but the issue with this solution is that we lose the benefits of being able to use the `first`, `last`, `after` variables in queries.

I get that we could add them back in, but this seems like unnecessary code duplication. Is there a way to avoid that? And if we have to duplicate, where do we duplicate from?

**Comment by jckw** [comment]
Actually, `first` and and `last` work, but using `after` has no effect.

**Comment by jckw** [comment]
Update: I realised what I needed to do was allow the `first`, `after` etc. params as default None to the function and not touch them. The rest is taken care of by Graphene automatically and there's no need for code duplication. 🎉

I also realised that I was trying to use node IDs for the "after" param instead of the cursor (which have different types), hence my last comment. 

In case anyone runs into something similar in the future, a working solution for my problem above is (very similar to @jkimbo's):

```
class Query(ObjectType)
    posts = DjangoFilterConnectionField(Post, some_field=graphene.String())

    def resolve_posts(root, info, some_field, first=None, last=None, after=None, before=None):
        return DjangoPost.objects.filter(...)
```

(where DjangoPost is just the Post object's model).

The filterset class can be removed entirely in my case (since I was just using it to perform the filtering done in resolve_posts now instead; or it can be kept to do additional filtering!

Thanks for the help!
